kubernetes-templates/04_cvat_frontend_deployment.yml

```diff
@@ -25,5 +25,5 @@ spec:
           image: openvino/cvat_ui:v1.2.0
           imagePullPolicy: Always
           ports:
-          - containerPort: 80
+            - containerPort: 80
```

Comment:
Please, look at `` kubernetes-templates/*``, I believe you didn't change these files.

Reply:
Looks like pre-commit hook autocorrected the files, we shall revert.